### DISCUSSION TITLE
Option for soft wrap indicator to be aligned with line number

### DISCUSSION THREAD

**Discussion by aquifolly** [helix-editor/helix#6887] [Ideas]
Similar to how Kate does it

**Comment by pascalkuthe** [comment]
can you post a screenshot it's not clear to me what you mean

**Reply by aquifolly** [reply]
![image](https://user-images.githubusercontent.com/35699052/234669097-3623c0f8-0e3a-4f11-8d57-9e998ead2a45.png)
